pandas-stubs/io/parsers/readers.pyi

```diff
@@ -44,7 +48,7 @@ def read_csv(
     | npt.NDArray
     | Callable[[str], bool]
     | None = ...,
-    dtype: DtypeArg | None = ...,
+    dtype: DtypeArg | defaultdict[str, Dtype] | None = ...,
```

Comment:
doesn't this have to be changed for all the overloads in `read_csv()` ?

Reply:
Probably, and also for read_clipboard.

Reply:
Fixed.

Reply:
I'm still seeing that only one of the overloads was changed.